### ISSUE TITLE
Incomplete install with pip using leo-6.2 from GH

### ISSUE THREAD

**Issue by dalito** [leo-editor/leo-editor#1546]
Hi,

I tried to install leo-6.2 from the zip-file downloaded from the GH-releases. After unpacking I ran 
```
C:\dev_src\leo-editor-6.2>pip install .
```
which resulted in an unusable installation. Leo starts but is not usable because many required non-python files are not installed. Here is the log:
```
Leo Log Window
Leo 6.2 final
Python 3.8.2, PyQt version 5.12.7
Windows 7 AMD64 (build 6.1.7601) SP1
current dir: C:/dev_src/leo-editor-6.2
load dir: C:/dev_src/_pyve/dev_py38/lib/site-packages/leo/core
global config dir: None
home dir: C:/Users/dlinke
No @menu setting found
file not found: C:/dev_src/_pyve/dev_py38/lib/site-packages/leo/doc/CheatSheet.leo
```
I also created a wheel for leo and tried to install using the wheel:
```
C:\dev_src\leo-editor-6.2>pip wheel .
C:\dev_src\leo-editor-6.2>pip install leo-6.2.final-py3-none-any.whl
```
This also resulted in an incomplete installation (which is not surprising).

The problem is not present if I do an editable install from the unzipped folder by
```
C:\dev_src\leo-editor-6.2>pip install -e .
```
So I can install and use leo this way, but the standard pip install should probably work too. At least it is suggested in the [documentation](http://leoeditor.com/installing.html#installing-from-sources-windows).

**Comment by edreamleo** [comment]
On Fri, Mar 27, 2020 at 9:54 AM David Linke <notifications@github.com>
wrote:

> I tried to install leo-6.2 from the zip-file downloaded from the
> GH-releases. After unpacking I ran
>
> C:\dev_src\leo-editor-6.2>pip install .
>
> Matt hasn't created Leo's pip package yet, because I have to release to GH
first.

Please be patient.

Edward

**Comment by dalito** [comment]
The zip-file from the github-release page for leo-6.2 misses the MANIFEST.in file. If I add this file pip install works fine.

**Comment by maphew** [comment]
Thanks for the report. From your comment figuring out why manifest is missing will fix #1512 also. I'm on it.

**Comment by maphew** [comment]
Found it! This is a regression of a bug first fixed in e82c341 (via #968) context diff . Edward the [MANIFEST.in history](https://github.com/leo-editor/leo-editor/commits/44e62a44179b0140d3381a47a36f9073a5aff3ba/.gitattributes shows you updated this file in 7dd1988e and reverted this fix.

Reading my comment form back then I think I see why. I said "Ignoring this breaks ..." , followed by a commented out line, and a comment marker means ignore. However in this instance we need it disabled. ... Yeah, the double negative made a mess of things. ;-)  This time I've just removed all of it. What can be more clear than nothing (in the way)!

Also removed mention of the commented out setup.py and setup.cfg. Those files also need to be exported for pip to function.

**Comment by maphew** [comment]
This is worth a v6.2.1 release.

Incidentally, there is a 5 file difference between the zip release of v6.2 and a brand new git clone of same, and a 22 file difference between my working copy clone of v6.2 and the release zip. I learned this by comparing each of these:
~~~
wget https://github.com/leo-editor/leo-editor/archive/v6.2.zip
git clone --branch v6.2 --depth=1 https://github.com/leo-editor/leo-editor.git

# adjust for local circumstance:
pushd ~/code/leo-editor && git checkout v6.2
~~~

Even a `git reset --hard` doesn't make them identical. The differences are explained by examining .gitattributes and .gitignore and their contents. I didn't realize before that a hard reset doesn't give a truly virgin environment. (Well, it could if .gitignore was empty, but that feature us there for a reason.)

**Comment by edreamleo** [comment]
On Sun, Mar 29, 2020 at 12:28 AM matt wilkie <notifications@github.com>
wrote:

> Found it! This is a regression of a bug first fixed in e82c341
> <https://github.com/leo-editor/leo-editor/commit/e82c3416c903ec47c8091cb5e0012d9b2d020d8b>
> (via #968 <https://github.com/leo-editor/leo-editor/issues/968>) context
> diff . Edward the [MANIFEST.in history](
> https://github.com/leo-editor/leo-editor/commits/44e62a44179b0140d3381a47a36f9073a5aff3ba/.gitattributes
> shows you updated this file in 7dd1988
> <https://github.com/leo-editor/leo-editor/commit/7dd1988e6eb82ea6dcf2a037d52160416486022a>
> and reverted this fix.
>
Oops. Sorry about that.

> Reading my comment form back then I think I see why. I said "Ignoring this
> breaks ..." , followed by a commented out line, and a comment marker means
> ignore. However in this instance we need it disabled. ... Yeah, the double
> negative made a mess of things. ;-) This time I've just removed all of it.
> What can be more clear than nothing (in the way)!
>
> Also removed mention of the commented out setup.py and setup.cfg. Those
> files also need to be exported for pip to function.
>
Good. I agree that 6.2.1 is needed.

Edward
